client/app/pod/queue.js

```diff
@@ -1,9 +1,10 @@
 angular.module('throwcast.queue', ['ngRoute'])
 
-.config(function ($routeProvider) {
+.config(function ($routeProvider, $sceProvider) {
   $routeProvider
   .when('/queue', {
     templateUrl: 'app/pod/queue.html',
     controller: 'QueueController',
   });
+  $sceProvider.enabled(false);
```

Comment:
Leave a comment about this being only for development, and replacing this with specific filters.

```
// TODO: ...
```